### PR TITLE
Fix/timer and bar color dw

### PR DESCRIPTION
**Pull request by devonwigle**
## Branch title

-Fix/timer and bar color dw

## Changes made

-  add css rule to allow scrolling
-  add html section to attach css rule
-  Fix timer no-reset bug
-  fix bar color bug

## Anything Refactored

- no

## Anything needed

- more

## Contributors

* Stevie Ambroise
* Alex Fritz
* Devon Wigle

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -336,6 +336,11 @@ aside {
   width: 40px;
 }
 
+.scroll-box {
```
Comment (alexmfritz) [inline styles.css:339]: SCROLLING!!!!

File: src/main.js [hunk 2]
```diff
@@ -210,8 +210,21 @@ function refreshTimer() {
   hide(logActivityButton, 'visibility')
 }
 
+function changeColorBar() {
```
Comment (alexmfritz) [inline src/main.js:213]: This was a great loop Devon!!!!

File: src/Activity.js [hunk 3]
```diff
@@ -1,31 +1,32 @@
...
-       show(startTimerButton, 'disabled');
-       startTimerButton.innerText = 'FINISHED!';
-       show(logActivityButton, 'visibility');
-       return timerDisplay.innerText = 'NICE JOB!';
-     }
-     minutes = minutes.toString().padStart(2, '0');
-     seconds = seconds.toString().padStart(2, '0');
-     timerDisplay.innerText = `${minutes}:${seconds}`}, 1000
-   )
+    disableButton(startTimerButton);
```
Comment (StevieAmb) [inline src/Activity.js:12]: Yes to fixing our timer!

**Review by StevieAmb — APPROVED** [review]

File: src/Activity.js [hunk 4]
```diff
@@ -9,23 +9,24 @@ class Activity {
...
+    var timer = setInterval(function () {
+      if(seconds > 0) {
+        seconds -= 1;
+      } else if (minutes > 0) {
+        minutes -= 1;
+        seconds = 59;
+      } else {
+        clearInterval(timer);
```
Comment (alexmfritz) [inline src/Activity.js:20]: The power of Google!

**Pull request merged**